Services/User/classes/class.ilUserUtil.php

```diff
@@ -227,7 +227,7 @@ public static function getPossibleStartingPoints($a_force_all = false)
         
         $all = array();
         
-        $all[self::START_PD_OVERVIEW] = 'overview';
+        $all[self::START_PD_OVERVIEW] = 'obj_dshs';
```

Comment:
"obj_dshs" is the translation of the "Dashboard Administration Node". 
I suggest to use the language variable "dash_dashboard" instead from language module 'dash'

Please also add $lng->loadLanguageModule('dash');

Reply:
ok,  I orientated my research to use a 'common' langvar, if possible.